settings.py

```diff
@@ -1,28 +1,48 @@
 import os
+# from tests.test_main import test_auth_user_models
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
+INSTALLED_APPS = [
+    "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+]
+
 # SECURITY WARNING: Modify this secret key if using in production!
 SECRET_KEY = "6few3nci_q_o@l1dlbk81%wcxe!*6r29yu629&d97!hiqat9fa"
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+AUTH_USER_MODEL = "db.User"
+
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
         "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
     }
 }
 
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.CommonPasswordValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.NumericPasswordValidator",
+    },
+]
+
 LANGUAGE_CODE = "en-us"
 
 TIME_ZONE = "Europe/Kiev"
 
 USE_I18N = True
 
 USE_TZ = False
```

Comment:
Setting `USE_TZ` to `False` disables timezone support. Ensure this aligns with your application's requirements, as it can affect datetime handling.